### DISCUSSION TITLE
Autcomplete With Routeledge Frequency Dictionary

### DISCUSSION THREAD

**Discussion by Danc2050** [psoukie/zipchord#166] [Ideas]
A friend of mine came up another option for ZipChord:
![image](https://user-images.githubusercontent.com/32185616/233644409-1b6d7ae5-dd3a-4087-a028-b6c80f6b522b.png)
Our is followed by one of these words 482025 times the corpus.
So you can have the text-expander work this way.
Let's say you type `our`
then the software will wait for the first letter of the colocate.
Our `o` entry will expand as `Our o<wn>` on screen with the `<wn>` indicating something you can accept with a tab. If you do not press tab and enter your characters instead, it will go away and you can enter a lower probable version.
[Routledge_Frequency_Dictionaries_Mark_Davies_Dee_Gardner_-_A_Frequency_Dictionary_of_Contemporary_American_English__Word_Sketches_Collocates_and_Thematic_Lists-Routledge_2010.pdf](https://github.com/psoukie/zipchord/files/11295994/Routledge_Frequency_Dictionaries_Mark_Davies_Dee_Gardner_-_A_Frequency_Dictionary_of_Contemporary_American_English__Word_Sketches_Collocates_and_Thematic_Lists-Routledge_2010.pdf)

**Comment by psoukie** [comment]
This is totally legitimate. My philosophy for ZipChord is that it is context independent, i.e. you can touch type without looking at any suggestions. Even with the inclusion of combined chords in v2.2, it focuses on something that you can memorize. (If the corpus is static, the typing is technically context independent, but impossible to memorize and predict, especially if you look at a history of a few words -- which you should to get better suggestions.)

**Reply by Danc2050** [reply]
Makes a lot of sense. I think that this feature is more prone to LCD screen typing. Thank you for considering it.
